libs/common/src/assets/i18n/ca.json

```diff
@@ -391,7 +393,7 @@
 	"stash.home.tags.title": "Gràfica de distribució de tags",
 	"stash.home.tags.total": "tags en total",
 	"stash.invited_user": "L'usuari {{user}} ha rebut un correu amb una invitació a participar a la caixa de coneixement",
-	"stash.kb_name_invalid": "El nom de la caixa de coneixement no és vàlid",
```

Comment:
I see this message is used in other parts of the app to validate the kb name field.
What do you think about adding another text like "stash.kb_slug_invalid" ?

Reply:
Good catch!
That being said, the invalidity of the name of the KB was always done based on the fact the name would become a slug right? shouldn't we remove this validation and display another field showing the generated slug ?
I feel like this is something to be reviewed with Anna as well though. So for now I'll just put back `stash.kb_name_invalid` and add a new one for slug validation.

Reply:
I don't remember why we did it this way (maybe easy of use), Anna knows it better than me.